src/Codeception/Module/Symfony.php

```diff
@@ -22,6 +22,10 @@
  *
  * * Maintainer: **raistlin**
  * * Stability: **stable**
+ * 
+ * ## Compatibility
+ *
+ * This module is compatible with all other Codeception modules, except WebDriver and the modules for other PHP frameworks.
```

Comment:
1. I didn't check if it exists.
  [Laravel5](http://codeception.com/docs/modules/Yii2#Parts)and Yii2 modules have ORM parts,
  [Yii2](http://codeception.com/docs/modules/Yii2#Parts) has 4 parts actually,
2. No, it wasn't the original purpose. I think that parts functionality was created to reduce the size of generated files. That's why REST module has Xml and Json parts.